src/Repositories/OrderRepository.php

```diff
@@ -155,8 +155,6 @@ public function createExactDeliveryAddressFromOrder(
         $address->City = $order->city ?? $order->customer->invoice_city;
         $address->Postcode = $order->postalcode ?? $order->customer->invoice_postalcode;
         $address->Phone = $order->mobile ?? $order->phone ?? $order->customer->mobile ?? $order->customer->telephone;
-        $address->AddressLine2 = $order->email ?? $order->customer->email;
```

Comment:
I understand that it can't be parsed, but don't they need this info in Exact? Maybe we should write this to another field? Or can we just leave this information out?

Reply:
It's added on the customer, I assume, and there are no such fields on the order